spec/index.html

```diff
@@ -52,14 +53,51 @@
   </script>
 
   <style>
+    body { color-scheme: light; }
+    body.darkmode { color-scheme: dark; }
+
+    /* Copy of hljs light/dark values, since hljs uses media
+     * prefers-color-scheme rule which isn't controlled by color-scheme. */
+    body .hljs {
+      --base: rgba(0, 0, 0, .03); /* Altered from hljs default of #fafafa; more like bikeshed/pygments */
+      --mono-1: #383a42;
+      --mono-2: #686b77;
+      --mono-3: #717277;
+      --hue-1: #0b76c5;
+      --hue-2: #336ae3;
+      --hue-3: #a626a4;
+      --hue-4: #42803c;
+      --hue-5: #ca4706;
+      --hue-5-2: #c91243;
+      --hue-6: #986801;
+      --hue-6-2: #9a6a01;
+    }
+    body.darkmode .hljs {
+      --base: #282c34;
+      --base: rgba(255, 255, 255, .05); /* Altered from hljs default of #282c34; more like bikeshed/pygments */
+      --mono-1: #abb2bf;
+      --mono-2: #818896;
+      --mono-3: #5c6370;
+      --hue-1: #56b6c2;
+      --hue-2: #61aeee;
+      --hue-3: #c678dd;
+      --hue-4: #98c379;
+      --hue-5: #e06c75;
+      --hue-5-2: #be5046;
+      --hue-6: #d19a66;
+      --hue-6-2: #e6c07b;
+    }
```

Comment:
This might lead to bugs in the future if respec/hljs is updated. I don't think we pin respec JS. I am going to open a ticket in ReSpec about it.

Reply:
@tpt That would be great! I failed to act on https://github.com/w3c/rdf-primer/pull/24 and haven't moved this CSS to rdf-common yet. But fixing it in ReSpec would be much better for all! (I probably should have gone that route from the beginning.)

(I might follow with what I've come up with re. highlighting, since ReSpec seems to handle that in a somewhat deprecated manner (CommonJS vs. ESM).)

Reply:
I removed the hljs styling. Until it is applied to ReSpec, example blocks will continue to have a white background in dark mode, but better not tie that style into the HTML on a one-off basis.

Reply:
I commented on https://github.com/speced/respec/issues/4871#issuecomment-2727318995 to mention the tentative solution for the Primer. Unless someone picks it up there, I might have some time for a PR on ReSpec, which as @Tpt says would be for the best (and the only robust solution).

Reply:
I also commented on https://github.com/speced/respec/issues/4871#issuecomment-2727512093 Sadly, fixing properly does not sound trivial.

Reply:
@tpt Important comment! (I agree it's not trivial, alas; but I think you're on to something there!)

Reply:
Aiming to solve this via https://github.com/speced/respec/pull/4925.